packages/talker_logger/lib/src/logger_io.dart

```diff
@@ -0,0 +1,147 @@
+import 'dart:io';
+
+import 'package:talker_logger/talker_logger.dart';
+
+class TalkerLogger {
+  TalkerLogger({
+    TalkerLoggerSettings? settings,
+    this.formatter = const ExtendedLoggerFormatter(),
+    LoggerFilter? filter,
+    void Function(String message)? output,
+  }) {
+    this.settings = settings ?? TalkerLoggerSettings();
+    // ignore: avoid_print
+    _output = output ??
+        (String message) => message.split('\n').forEach(stdout.writeln);
```

Comment:
The only difference between logger_io and logger_web is in this line
Please make outout_io.dart and output_web.dart files with **output(String message)** method realizations 

We can't make two almost identical files

Reply:
Hello @Frezyx 

As you mentioned, I've added only the changes to the two files and preserved most of the original code. The modified code runs well on both web and Android, but on Android, stdout.writeln did not work, so I changed it to print. Thank you.